package.json

```diff
@@ -20,15 +20,18 @@
     "dev": "./dev.sh"
   },
   "devDependencies": {
-    "@directus/extensions-sdk": "9.18.1",
+    "@directus/extensions-sdk": "9.24.0",
+    "@nuxt/types": "^2.16.3",
+    "@tsconfig/nuxt": "^1.0.0",
+    "@types/node": "^18.15.12",
     "@types/prismjs": "^1.26.0",
-    "axios": "^0.27.2",
-    "sass": "^1.55.0",
-    "typescript": "^4.8.4",
-    "vue": "^3.2.40"
+    "axios": "^1.3.6",
+    "sass": "^1.62.0",
+    "typescript": "^5.0.4",
+    "vue": "^3.2.47"
   },
   "dependencies": {
-    "@directus/shared": "^9.18.1",
+    "@directus/shared": "^9.24.0",
```

Comment:
I'm worried that if we use the directus shared types for a version after 9.0, then the directus host compatibility set at 9.0 might not actually be compatible.